configs/test_local.properties

```diff
@@ -25,3 +25,5 @@ test.ethereum.gasLimit=6000000
 # --------- Bitcoin ---------
 test.bitcoin.walletPath=deploy/bitcoin/test-btc.wallet
 test.bitcoin.blockStoragePath=deploy/bitcoin/blocks
+# --------- Timeout (minutes) ---------
+test.testTimeout=5
```

Comment:
I think we can hardcode this value.

Reply:
Agree.

Reply:
Done as a constant value In ConfigHelper